### PR TITLE
Add a template for several kinds of issues

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Signed-off-by: Slawomir Jankowski <slawomir.jankowski@intel.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Logs
+<!--- Error logs in any form are really helpful -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used (commit hash or tag):
+* Operation System:
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/bug.md:37]: Operating

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Steps to Reproduce
+<!--- Provide an repeatable set of steps to reproduce this bug -->
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/bug.md:19]: a repeatable

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+name: Enhancement proposal
+about: Create a enhancement proposal to help us improve
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/enhancement.md:3]: an enhancement

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+title: ''
+labels: enhancement
+assignees: ''
+---
+
+## Description
+<!--- Provide a more detailed description of wanted functionality -->
+
+## Reason
+<!--- Why you need this functionality? What's the gain of introducing it? -->
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/enhancement.md:13]: Why do you need

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Your Environment
+<!--- Include as many relevant details about the environment you're working in -->
+* Version used (commit hash or tag):
+* Operation System:
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/enhancement.md:18]: Operating

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+## Question
+<!--- Ask a question -->
+
+## Motivation
+<!--- Why are you asking? Is there any need other than curiosity? -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you're working in -->
+* Version used (commit hash or tag):
+* Operation System:
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/question.md:18]: Operating

File: .github/ISSUE_TEMPLATE/bug.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Logs
+<!--- Error logs in any form are really helpful -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used (commit hash or tag):
```
Comment (Deixx) [inline .github/ISSUE_TEMPLATE/bug.md:36]: to leave no room for ambiguity - add OpenCAS ;)

**Review by Deixx — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
```
Comment (katlapinka) [inline .github/ISSUE_TEMPLATE/bug.md:3]: improve what? ;)

File: .github/ISSUE_TEMPLATE/question.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+assignees: ''
+---
+
+## Question
+<!--- Ask a question -->
+
+## Motivation
+<!--- Why are you asking? Is there any need other than curiosity? -->
+
+## Your Environment
```
Comment (katlapinka) [inline .github/ISSUE_TEMPLATE/question.md:15]: This is not obligatory for questions I think - maybe add this information

File: .github/ISSUE_TEMPLATE/bug.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+## Context
+<!--- How has this bug affected you? What were you trying to accomplish? -->
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Logs
+<!--- Error logs in any form are really helpful -->
+
+## Your Environment
```
Comment (katlapinka) [inline .github/ISSUE_TEMPLATE/bug.md:34]: I think you can suggest to place here for example output of 'casadm -P', 'casadm -L' and 'lsblk' commands

File: .github/ISSUE_TEMPLATE/bug.md (hunk 8 above)
Comment (Ostrokrzew) [reply .github/ISSUE_TEMPLATE/bug.md:3]: OK, You've got me, I copied that from GH's default bug report template.

**Review by Deixx — DISMISSED** [review]

**Review by katlapinka — APPROVED** [review]

**Review by Deixx — APPROVED** [review]

**Pull request merged**